irs-edc-client/src/main/java/org/eclipse/tractusx/irs/edc/client/policy/Permission.java

```diff
@@ -35,10 +38,12 @@
 @Getter
 @AllArgsConstructor
 @NoArgsConstructor
+@Builder
 public class Permission {
 
+    @Schema(implementation = PolicyType.class, example = "ACCESS")
```

Comment:
can we use "USE" instead because that leads more to the use as access is directly handled by MIW and EDC. ACCESS policies are not used in this context.